### PR TITLE
added report for school list

### PR DESCRIPTION
**Pull request by aman-tunerlabs**

### CODE REVIEW

**Comment by aman-tunerlabs** [comment]
{{url}}/assessment/api/v1/reports/schoolList?programId=PROGID01&componentId=5b98fa069f664f7e1ae7498c 

programId - submissions programExternalId
componentId - submissions evaluationFrameworkId

File: controllers/v1/reportsController.js [hunk 1]
```diff
@@ -2681,6 +2678,106 @@ module.exports = class Reports {
...
+        if (!componentId) {
+          throw "Component id is missing"
+        }
+
+        let submissionQueryObject = {
+          programExternalId:programId,
+          evaluationFrameworkId:componentId
+        }
+
+        let submissionsIds = await database.models.submissions.find(
```
Comment (aks30) [inline controllers/v1/reportsController.js]: Why do we need to get school list from submissions ? @aman-tunerlabs

**Comment by aman-tunerlabs** [comment]
{{url}}/assessment/api/v1/reports/schoolList/PROGID01?componentId=5b98fa069f664f7e1ae7498c

few changes in endpoint 
PROGID01 - from programs collection externalId
componentId - from programs collection components id

**Comment by aman-tunerlabs** [comment]
{{url}}/assessment/api/v1/reports/schoolList/PROGID01?componentId=EF-DCPCR-2018-001

where componentId - evaluationFramework external id.

**Pull request merged**